docs/frontend-system/building-apps/01-index.md

```diff
@@ -0,0 +1,124 @@
+---
+id: index
+title: Building Frontend Apps
+sidebar_label: Overview
+# prettier-ignore
+description: Building frontend apps using the new frontend system
+---
+
+> **NOTE: The new frontend system is in alpha and is only supported by a small number of plugins.**
+
+To get set up quickly with your own Backstage project you can create a Backstage App.
+
+A Backstage App is a monorepo setup that includes everything you need to run Backstage in your own environment.
+
+## Creating a new app
+
+To create a Backstage app, you will need to have Node.js Active LTS Release installed.
+
+The easiest way to run the create app package is with `npx`:
+
+```sh
+npx @backstage/create-app@latest
+```
+
+This will create a new Backstage App inside the current folder. The name of the app-folder is the name that was provided when prompted.
+
+:::note
+The created app will currently be templated for use in the legacy frontend system, and you will need to replace the existing app wiring code.
+:::
+
+## The app instance
+
+The starting point of a frontend app is the `createApp` function, which accepts a single options object as its only parameter. It is imported from `@backstage/frontend-app-api`, which is where you will find most of the common APIs for building apps.
+
+This is how to create a minimal app:
+
+```tsx title="in src/index.ts"
+import ReactDOM from 'react-dom/client';
+import { createApp } from '@backstage/frontend-app-api';
+
+// Create your app instance
+const app = createApp({
+  // Features such as plugins can be installed explicitly, but we will explore other options later on
+  features: [catalogPlugin],
+});
+
+// This creates a React element that renders the entire app
+const root = app.createRoot();
+
+// Just like any other React we need a root element. No server side rendering is used.
+const rootEl = document.getElementById('root')!;
+```
+
+Note that the `createRoot` exports an element instead of a component and it no longer accepts a custom root element as parameter, so this is the reason you can pass the returned element directly to React `createRoot`. See [customizing your root app element](#app-root) for more details on how to do it with extensions.
```

Comment:
This is already covered in the app migration docs, do you think it would be alright to just explain that `createRoot` returns an element that you pass to `render` instead? Just in the interest of keeping these docs as long-lived as possible

Reply:
Done